### DISCUSSION TITLE
Some questions about the rotation case and input of FLOWUnsteady

### DISCUSSION THREAD

**Discussion by 12029904** [byuflowlab/FLOWUnsteady#210] [Q&A]
Dear FLOWUnsteady team and author

A while ago, I accidentally came into contact with FLOWUnsteady and was deeply impressed by its astonishing policy speed and accuracy. I am a CFD practitioner and mainly use Fluent as a simulation tool, so I am not particularly familiar with FLOWUnsteady and its language Julia. Recently, I am working hard to learn related knowledge.

However, I encountered some issues while running the built-in case of FLOWUnsteady. For simple wing models, the case can run successfully, but in examples with rotating boundaries such as Propeller and Rotorhover, errors always occur during operation, as shown in the following figure

![1](https://github.com/user-attachments/assets/137a8b0e-5433-4615-80f0-7fb693453753)
![2](https://github.com/user-attachments/assets/e2971898-4059-4617-9979-430ee81fc20b)
![3](https://github.com/user-attachments/assets/50c2d07e-257e-4201-b97a-1b029e154546)

After the software was installed, no configuration files were modified. I don't know if it was a problem with where I set it up or for some other reason. If you could tell me how to overcome this problem, I would be very grateful.

In addition, as far as I know, geometric models need to be segmented into multiple sections and input into the software in CSV format, which poses a significant obstacle to the promotion of the software (preparing the model will take a long time). I don't know if it's possible to directly use STL or NAS format surface mesh files as input for software simulation calculations. If possible, that would be great.

Looking forward to your reply.
Best Regards.

**Comment by nrnaveenkumr** [comment]
Hi @12029904 ,  can you share your "propeller.jl" file and other supporting files for better understanding
